src/lib/github/addIssuesToRepo.ts

```diff
@@ -11,21 +12,33 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
   const octokit = context.github;
   const { owner, repo } = getBasicRepoProps (context);
   const ghIssues = await getAllRepoIssues(context);
+
   newIssues.forEach(async ({title, body, authors}) => {
     const assignees = uniq(authors);
+
     const currentGHIssue = find(ghIssue => ghIssue.title === title, ghIssues);
+    const newGHIssueComments = currentGHIssue.comments.includes("- [x]") || !(currentGHIssue.comments === body)
```

Comment:
`!(currentGHIssue.comments === body)` this equality comparison is wrong.
- type for `currentGHIssue.comments` is `string[]`
- type for `body` is `string`
```suggestion
    const newGHIssueComments = currentGHIssue.comments.includes("- [x]")
```

Reply:
@epicallan , i modified the `type` for `body`, see [this](https://github.com/freelyformd/github-todos/blob/bf100cb85d7720c848adcf6a304cb6fa839bcc77/src/lib/github/types.ts#L11)